lib/kramdown/parser/govuk.rb

```diff
@@ -53,17 +53,110 @@ def add_link(element, href, title, alt_text = nil, ial = nil)
       def parse_block_html
         return false if CUSTOM_INLINE_ELEMENTS.include?(@src[1].downcase)
 
+        super
+      end
+
+      # We override the parse_list method with a modified version where ordered lists are
+      # disabled (for use with legislative lists). The majority of the method body is copied
+      # over, only the LIST_START is changed. Previously, we dynamically modified some of the
+      # class-scoped variables used in this method; this provides a thread-safe alternative.
```

Comment:
How would you feel about including a link to Github for the version of the method you copied in this comment? (e.g. maybe it was this one https://github.com/gettalong/kramdown/blob/REL_2_3_1/lib/kramdown/parser/kramdown/list.rb#L54 ?)

Reply:
Done 👍